### ISSUE TITLE
Icons disappeared

### ISSUE THREAD

**Issue by Shogun1978** [resoai/TileBoard#574]
Since today I don't see any icons. Tried Chrome, Firefox and Edge, same result on all three. Also the icons of homepage and second page are gone. Is there an issue with materialdesignicons.com being down?

Another question: what is the best way to to generate a valid and checked config.js? I always struggle with a missing comma or brackets because the config file get's pretty huge.

**Comment by rchl** [comment]
Latest releases of Tile board don't really on fetching icons from remote servers so this shouldn't be possible.

**Comment by Shogun1978** [comment]
Thank you for the fast reply. I use tileboard in a docker container (fbrinker/tileboard from docker hub). Worked great but since today: no icons. I copied the configuration to another docker host for testing before moving to the newest release. When I want to rebuild the setup with the newest version (downloaded and extracted Tileboard.zip and copied my config.js to the testing host), I get the following errer message:

If you see this page then you you have most likely picked a wrong way to install TileBoard.

If you are not a TileBoard developer and just want to use it then go to Releases page and download the TileBoard.zip file attached to the latest release.

You can also check out How To Use instructions for more detailed instructions.

Any ideas what I'm doing wrong? I was also not able to run tileboard on the home assistant systeme (supervised install with https certs). That was the reason for moving to a docker image.

**Comment by rchl** [comment]
Maybe something broke in that container then and you should check with the author of that container.

As I said, the icons are bundled with Tileboard.

You can check the network inspector in your browser to check if there any failing network requests.

**Comment by Shogun1978** [comment]
Could be right. I contacted the author, it's an automated build. Maybe there is something not ok. Parallel I tried just setting up tileboard on my testing HA system in the www folder. Funny: it works. The most funny thing: on my tablet in the livingroom, the old setup (with the old container) shows the icons, my local laptop doesn't. Even after clearing the cache... seems like a 2020-bug here in my apartment. I will close the issue... I can't explain this behavior, when it's working on the tablet but not the laptop.
